docs/conf.py

```diff
@@ -30,7 +30,7 @@
 # -- General configuration ------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-needs_sphinx = '1.4'
+#needs_sphinx = '1.4'
```

Comment:
Not that I know of. We have control over the sphinx version used to build the docs in our containthedocs image, so we don't have to worry about it too much on a project level.